src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -604,6 +605,7 @@ private function addAnnotationsSection(ArrayNodeDefinition $rootNode)
                     ->info('annotation configuration')
                     ->canBeDisabled()
```

Comment:
No, I can't because I still need the logic inside `canBeDisabled` but I'm simply overriding the boolean node and its default value.

However, if it seems less "hacky", I can replace the fix by:

```diff
diff --git a/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php b/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php
index 4b9af8c..4a7dfa2 100644
--- a/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php
+++ b/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php
@@ -603,9 +603,8 @@ class Configuration implements ConfigurationInterface
             ->children()
                 ->arrayNode('annotations')
                     ->info('annotation configuration')
-                    ->canBeDisabled()
+                    ->{class_exists(Annotation::class) ? 'canBeDisabled' : 'canBeEnabled'}()
                     ->children()
-                        ->booleanNode('enabled')->defaultValue(class_exists(Annotation::class))->end()
                         ->scalarNode('cache')->defaultValue('php_array')->end()
                         ->scalarNode('file_cache_dir')->defaultValue('%kernel.cache_dir%/annotations')->end()
                         ->booleanNode('debug')->defaultValue($this->debug)->end()

```